endpoints/remotehost/remotehost

```diff
@@ -357,10 +355,12 @@ function launch_osruntime() {
             do_ssh $user@$host mkdir -p $container_mount/tmp
             do_ssh $user@$host mount --verbose --options bind $remote_data_dir $container_mount/tmp
             echo "Adding host directories to container mount"
-            for fs in proc dev dev/hugepages proc sys lib/modules usr/src boot; do
-                echo "Adding $fs"
+            for fs in ${chroot_rbind_mounts}; do
+                echo "Adding ${fs} to ${container_mount}"
                 do_ssh $user@$host mkdir -p $container_mount/$fs
-                do_ssh $user@$host mount --verbose --options bind /$fs $container_mount/$fs
+                do_ssh $user@$host mount --verbose --options rbind /$fs $container_mount/$fs
+                do_ssh $user@$host mount --verbose --make-rslave $container_mount/$fs
```

Comment:
I eventually figured out that the `--make-rslave` mount option was needed to "fixup" some properties of the `rbind` mounts so that the eventual recursive `umount` would succeed.  This seems to the magic that makes the `mount`/`umount` work when `rbind` is being used.